internal/pkg/platform/activity_test.go

```diff
@@ -147,37 +148,37 @@ func TestPlatformActivity_StreamingLogs(t *testing.T) {
 				}()
 				return ctx
 			},
-			ExpectedAsserts: func(t *testing.T, cm *shared.ClientsMock) {
+			ExpectedAsserts: func(t *testing.T, ctx context.Context, cm *shared.ClientsMock) {
 				// with the above polling/canceling setup, expectation is activity called only once.
 				cm.ApiInterface.AssertNumberOfCalls(t, "Activity", 1)
 			},
 		},
 	} {
 		t.Run(name, func(t *testing.T) {
-			ctx := context.Background()
-			ctx = context.WithValue(ctx, config.CONTEXT_TOKEN, "sometoken")
 			// Create mocks
+			ctxMock := slackcontext.MockContext(context.Background())
+			ctxMock = context.WithValue(ctxMock, config.CONTEXT_TOKEN, "sometoken")
```

Comment:
note: One of the few unique changes, where the activity test modifies a value in the context.